cmd/main_test.go

```diff
@@ -267,3 +275,38 @@ var _ = AfterSuite(func() {
 		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Error killing the server during test teardown: %v", err))
 	}
 })
+
+func GetNamespace(k8sClient client.Client, namespace string) (k8sapi.Namespace, error) {
```

Comment:
Is there a way for us to create namespace objects instead of asking the server for them?

Reply:
We can update createNamespace function to return the namespace object but since you mentioned not to modify it, I have written separate function to get namespace.

Reply:
It's a test function, you can modify it as long as it doesn't affect the already existing code. 
but since it is called from the beforesuite, how will you get its return value